apps/hash-graph/lib/graph/src/subgraph/edges/edge.rs

```diff
@@ -1,16 +1,48 @@
-use serde::Serialize;
+use serde::{ser::SerializeStruct, Serialize, Serializer};
 use utoipa::{openapi, ToSchema};
 
-#[derive(Debug, Hash, PartialEq, Eq, Serialize)]
-#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[derive(Debug, Hash, PartialEq, Eq)]
 pub struct OutwardEdge<K, E> {
     pub kind: K,
     /// If true, interpret this as a reversed mapping and the endpoint as the source, that is,
     /// instead of Source-Edge-Target, interpret it as Target-Edge-Source
```

Comment:
Needs updating

Reply:
Thanks, removed in [Remove outdated comment](https://github.com/hashintel/hash/pull/2230/commits/7a2c56ddc8fbb51f0baedeb2afaadd757fd3e7b4) and added a doc string in [Add comment to EdgeDirection](https://github.com/hashintel/hash/pull/2230/commits/ddfa958cd8ef57d14382f4f3a009d187a30b3392)